change/@microsoft-teams-js-f4f055db-c85c-4ea2-b470-278759a1aafb.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "In `dialog.ts` moved `Resize` functionality in a subcapability named `update`.",
```

Comment:
I think it would be good to specify the capability since the client would be calling it from the capability rather than looking at the file name. So maybe something like:

"`dialog.resize()` function has been moved to a new dialog subcapability and is now `dialog.update.resize()`."